.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: android
+
+dist: trusty
+
+install: true
+
+jdk:
+  - oraclejdk8
+
+before_install:
+  - yes | sdkmanager "platforms;android-30"
+  - yes | sdkmanager "build-tools;30.0.2"
+
+script:
+  - ./gradlew build
```

Comment:
Typically tests are not included in the build by default. In this case, you should use task `check`, or include tests

Reply:
Tests are included (lint is included as well). Documentation says:

> Assembles and **tests** this project.

Also checked actual build logs and also with `./gradlew build --dry-run`.